### PR TITLE
Remove Polymer.GestureEventListeners

### PR DESCRIPTION
**Pull request by limonte**
The same as https://github.com/vaadin/vaadin-text-field/pull/108

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-button/44)

### CODE REVIEW

**Comment by manolo** [comment]
:lgtm_strong: 

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-button/44#-:-KtQifjbmmRTMyIx9GRx:b-uwkowd)*

File: vaadin-button.html [hunk 1]
```diff
@@ -42,7 +41,7 @@
       throw new Error(`Unexpected Polymer version ${Polymer.version} is used, expected v2.0.0 or later.`);
     }
 
-    (function() {
+    {
```
Comment (abdonrd) [inline vaadin-button.html]: Why remove the `function`?

Comment (limonte) [reply vaadin-button.html]: http://wesbos.com/es6-block-scope-iife/

Comment (abdonrd) [reply vaadin-button.html]: Oh, thanks @limonte!

Comment (limonte) [reply vaadin-button.html]: You're always very welcome @abdonrb!

**Pull request merged**